docs/integrations/sources/instagram-migrations.md

```diff
@@ -0,0 +1,9 @@
+# Instagram Migration Guide
+
+## Upgrading to 2.0.0
+
+This release adds a default primary key for the streams UserLifetimeInsights and UserInsights, and updates the format of timestamp fields to include timezone information.
```

Comment:
This release adds a default primary key for the streams UserLifetimeInsights and UserInsights, and updates the format of timestamp fields in the Media and Stories streams to include timezone information.

Reply:
Fixed